config/locales/views.en.yml

```
@@ -187,7 +187,7 @@ en:
       confirmed: 'Thank you for your registration. You''re now logged in.'
     unlocks:
       send_instructions: 'In a few minutes, you''ll receive an e-mail with instructions how to unlock your account.'
-      unlocked: 'Your account has been unlocked. You''re now logged in.'
+      unlocked: 'Your account has successfully been unlocked.'
```

```suggestion
      unlocked: 'Your account has been successfully unlocked.'
```
Aber Änderungen an den nicht-deutschen Übersetzungen würde ich eher im Transifex machen. Wenn du sie im Code machst, musst du nach dem mergen von Transifex pullen, die Übersetzungs-Changes erneut anwenden, und wieder nach Transifex pushen.

---

Bedeutet, jeweils die deutschen Übersetzungen im Code einchecken und den Rest im Transifex handeln?
Was wenn ich eine neue Translation anlege? Auf Deutsch korrekt übersetzen und in den anderen Files nur den Key mit leerer Value?

---

Meine Empfehlung wäre, immer nur mit deutschen Übersetzungen zu arbeiten. Den Rest bringt Transifex dann hin, auch neue Translation Keys. Wenn man weiss was man macht kann man aber natürlich wo nötig / sinnvoll davon abweichen.